components/ai_chat/core/common/mojom/ai_chat.mojom

```diff
@@ -193,8 +193,31 @@ struct LeoModelOptions {
   uint32 long_conversation_warning_character_limit;
 };
 
+/**
+  * context_size:
+  *   This value is provided by the user when adding/configuring
+  *   a custom model. If not provided, Brave will default to using
+  *   ai_chat::kCustomModelDefaultContextSize.
```

Comment:
optional nit: you could comment these directly over the members? Not sure if we have a convention here though, so completely optional

Reply:
I usually do place comments just above members as many editors will promote these to intellisense in remote references. Will do that here as well.

Reply:
That's what I thought might happen in mojom, but it doesn't :D I had a bit of a dig into the generated code but the code comments just get dropped :laughing: